exporter/collector/README.md

```diff
@@ -54,7 +54,7 @@ These instructions are to get you up and running quickly with the GCP exporter i
           exporters: [googlecloud, logging]
         metrics:
           receivers: [otlp]
-          processors: [memory_limiter]
+          processors: [memory_limiter, batch]
```

Comment:
Why are we adding `batch` here? I thought that we were doing batching internally so we didn't need this.

Reply:
We are still splitting large batches in the exporter, but aren't grouping smaller batches together anymore after this change.  We still need the processor to do that for us.

Reply:
Gotcha. It does feel a little bit icky to have two batching steps, but correctness > simplicity so I guess that's fine.

Reply:
Its what we do in the opencensus exporter as well, so at least it won't be a regression for anyone.